bundle/manifests/hw-event-proxy-operator.clusterserviceversion.yaml

```diff
@@ -239,9 +239,7 @@ spec:
           - watch
         serviceAccountName: hw-event-proxy-operator-controller-manager
       deployments:
-      - label:
-          control-plane: controller-manager
```

Comment:
any functional impact for removing this label?

Reply:
This is an artifact of an update made when the designer had operator-sdk 1.19.0 in their environment, rather than the expected version, a scenario which this PR will address. Removing this label should have no impact on the deployment